Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -295,3 +380,19 @@ extension SecureEnclaveValet {
     }
 
 }
+
+// MARK: - Private Extensions
+
+private extension SecureEnclaveAccessControl {
```

Comment:
Nit: let's attach the access control to the property below rather than attaching it here. I prefer to keep access control both explicit and local.
```suggestion
extension SecureEnclaveAccessControl {
```

Reply:
Sounds good 👍